React/Views/RCTComponentData.m

```diff
@@ -439,11 +423,14 @@ - (void)setProps:(NSDictionary<NSString *, id> *)props forShadowView:(RCTShadowV
     }
   }
 #endif
-
+  
+  Class superClass = [_managerClass superclass];
+  
   return @{
     @"propTypes": propTypes,
     @"directEvents": directEvents,
     @"bubblingEvents": bubblingEvents,
+    @"baseModuleName": superClass != [NSObject class] ? [self moduleNameForClass:superClass] : [NSNull null]
```

Comment:
nit: Please avoid "negative logic": Changing `!=` to `==` will make this code easier to read.